### ISSUE TITLE
As a user, I need clearer formatting between businesses

### ISSUE THREAD

**Issue by lindseyberk** [heartdeco/addison-county-community-table#14]
On the Participating businesses and restaurants lists, each business should stand out more - can you add space or a line/graphic between each business? Or will logos be added?

**Comment by justindoran** [comment]
Agreed. We're going to be adding uploadable logo images. That involves contracting with a separate image hosting service for performance/cost reasons, so we've pushed it off to the next sprint for now. We'll update this issue when we have a timeline for it.
